### DISCUSSION TITLE
PaddleX/latest/module_usage/instructions/model_python_API

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleX#2657] [General]
# PaddleX/latest/module_usage/instructions/model_python_API

https://paddlepaddle.github.io/PaddleX/latest/module_usage/instructions/model_python_API.html

**Comment by linqi1321** [comment]
save_to_xlsx()方法报错：AttributeError: 'TopkResult' object has no attribute 'save_to_xlsx'. Did you mean: 'save_to_img'?

**Reply by cuicheng01** [reply]
请问是什么模型呢？

**Reply by z666-123** [reply]
我也是这个报错：Traceback (most recent call last):
  File "/root/work/test_api/test_paddlex.py", line 8, in <module>
    res.save_to_xlsx("./output/res.xlsx")
AttributeError: 'DetResult' object has no attribute 'save_to_xlsx'. Did you mean: 'save_to_img'?。代码是：```from paddlex import create_model
model = create_model(model_name="RT-DETR-L_wired_table_cell_det")
output = model.predict("./test_imgs/1743578240865.jpg", threshold=0.3, batch_size=1)
for res in output:
    res.print(json_format=False)
    res.save_to_img("./output/")
    res.save_to_json("./output/res.json")
    res.save_to_xlsx("./output/res.xlsx")
    # save_to_xlsx()
```

**Reply by z666-123** [reply]
请问如何调整save_to_img方法，使得在输出的图像中，cell和score字体变小
